mintpy/tsview.py

```diff
@@ -111,6 +129,42 @@ def cmd_line_parse(iargs=None):
     if not inps.fig_size:
         inps.fig_size = [8.0, 4.5]
 
+    # temporal model fitting
+    if inps.model_bool:
+        # --exp option: convert cmd inputs into dict format
+        inps.expDict = dict()
+        if inps.exp:
+            for exp_list in inps.exp:
+                onset_time, char_times = exp_list[0], exp_list[1:]
+                if len(onset_time) == 8:
+                    if len(char_times) > 0:
+                        inps.expDict[onset_time] = np.array(char_times).astype(float).tolist()
+                    else:
+                        msg = 'NO characteristic time found: {}\n'.format(char_times)
+                        msg += 'one or more characteristic time(s) are required for each onset date for the exp function, e.g.:\n'
+                        msg += '--exp 20181026 60 OR\n'
+                        msg += '--exp 20161231 80.5 200  # append as many char_times as you like!'
+                        raise ValueError(msg)
+                else:
+                    raise ValueError('input onset time is NOT in YYYYMMDD format: {}'.format(onset_time))
+
+        # --log option: convert cmd inputs into dict format
+        inps.logDict = dict()
+        if inps.log:
+            for log_list in inps.log:
+                onset_time, char_times = log_list[0], log_list[1:]
+                if len(onset_time) == 8:
+                    if len(char_times) > 0:
+                        inps.logDict[onset_time] = np.array(char_times).astype(float).tolist()
+                    else:
+                        msg = 'NO characteristic time found: {}\n'.format(char_times)
+                        msg += 'one or more characteristic time(s) are required for each onset date for the log function, e.g.:\n'
+                        msg += '--exp 20181026 60 OR\n'
+                        msg += '--exp 20161231 80.5 200  # append as many char_times as you like!'
+                        raise ValueError(msg)
+                else:
+                    raise ValueError('input onset time is NOT in YYYYMMDD format: {}'.format(onset_time))
+
```

Comment:
This part is duplicated from ts2vel.py. Could we move them, in ts2vel.py, to a function and call them here?

Reply:
Ok, that makes sense. Thanks